### PR TITLE
Error fix in allTubes.php

### PR DESCRIPTION
**Pull request by drull755**
[08-Sep-2015 09:47:45 UTC] PHP Warning:  Invalid argument supplied for foreach() in /opt/opoc/sites/beanstalk/lib/tpl/allTubes.php on line 23
[08-Sep-2015 09:47:45 UTC] PHP Stack trace:
[08-Sep-2015 09:47:45 UTC] PHP   1. {main}() /opt/opoc/sites/beanstalk/public/index.php:0
[08-Sep-2015 09:47:45 UTC] PHP   2. require_once() /opt/opoc/sites/beanstalk/public/index.php:46
[08-Sep-2015 09:47:45 UTC] PHP   3. require_once() /opt/opoc/sites/beanstalk/lib/tpl/ajax.php:1

### CODE REVIEW

File: lib/tpl/allTubes.php [hunk 1]
```diff
@@ -20,7 +20,7 @@
                     </tr>
                 </thead>
                 <tbody>
-                    <?php foreach ($tubes as $tubeItem): ?>
+                    <?php foreach ((is_array($tubes) ? $tubes : []) as $tubeItem): ?>
```
Comment (pentium10) [inline lib/tpl/allTubes.php]: @drull755 Please change this line to be PHP5.2 compatible as this library is still compatible with PHP5.2

**Comment by pentium10** [comment]
@drull755  were you able to do the changes?

**Comment by drull755** [comment]
Sorry, had no time. Will do it this week.

**Comment by drull755** [comment]
@pentium10 , done.

**Pull request merged**